public/youtube_producer/app-bundle.js

```diff
@@ -612,7 +594,7 @@ module.exports = function ($location, $http) {
 				maxResults: 10,
 				q: searchValue,
 				part: 'snippet',
-				key: token,
+				key: config.youtubeDeveloperToken,
 				type: 'playlist'
 			}
 		}).success(function(data){
```

Comment:
This file contains a lot of duplicate code that is already present in `clients/youtube_producer/js/mainTemplate.html`. Is this intentional?